lib/ecto/adapters/clickhouse/connection.ex

```diff
@@ -345,6 +347,9 @@ defmodule Ecto.Adapters.ClickHouse.Connection do
   defp join_qual(:inner), do: " INNER JOIN "
   defp join_qual(t) when t in [:inner_lateral, :array], do: " ARRAY JOIN "
   defp join_qual(t) when t in [:left_lateral, :left_array], do: " LEFT ARRAY JOIN "
+  # eh...
+  # > JOIN without specified type implies INNER. Keyword OUTER can be safely omitted. Alternative syntax for CROSS JOIN is specifying multiple tables in FROM clause separated by commas.
+  # from https://clickhouse.com/docs/en/sql-reference/statements/select/join
   defp join_qual(:left), do: " LEFT OUTER JOIN "
```

Comment:
We might also need to change the default to be the same as in ClickHouse. That would be a breaking change :)